### ISSUE TITLE
ResBlock

### ISSUE THREAD

**Issue by wizardk** [hcy71o/AutoVocoder#3]
Hi @hcy71o , I know in the paper it is said:
_Each such block consists of two 2D convolutional layers of width 3, followed by
a 2D batch norm and a ReLU nonlinearity. This basic block
also applies a residual, by summing the input with the output,
but only if the number of input channels is the same as the
number of output channels._

I want to know why residual link in ResBlock is used in each layer, even on BatchNorm2d and ReLU. Because as far as I know, we usually add the residual link after the whole block. For example:

```
def forward(self, x):
    res = x
    for c in self.convs:
        x = c(x)
    if self.out_ch == self.in_ch:
        x = res + x
    return x
```

**Comment by wizardk** [comment]
Moreover, I did an experiment and could not produce intelligible audio.

**Comment by redmist328** [comment]
Hi @wizardk, 
I have the same question with you. Do you work it out? 
I would be appreciate if you could tell me how you fix this and whether your experiments result is well.
